pyproject.toml

```diff
@@ -65,6 +69,12 @@ create = [
 openai = [
     "openai",
 ]
+rag = [
+    "ragas",
+    "unstructured",
+    "optuna",
+    "onnxruntime"
+]
```

Comment:
Let's extend this group to include all langchain and llama-index dependencies, otherwise this will fail after install

```suggestion
rag = [
    "ragas",
    "unstructured",
    "optuna",
    "onnxruntime",
    "langchain-cohere",
    "langchain-mistralai",
    "langchain-openai",
    "langchain",
    "llama-index"
]
```